airbyte-integrations/connectors/source-s3/source_s3/v4/stream_reader.py

```diff
@@ -157,7 +157,7 @@ def _page(
                 break
 
     def _handle_file(self, file):
-        if file["Key"].endswith("zip"):
+        if file["Key"].endswith(".zip"):
```

Comment:
what problem does this solve? did we encounter files ending in `zip` that did not have an extension?

Reply:
@girarda Yes, the sync failed due to the file without an extension, ending with 'zip'.